prototype_source/graph_mode_dynamic_bert_tutorial.rst

```diff
@@ -0,0 +1,551 @@
+
+Since we use per-channel quantization, we get per-channel scales tensor.
+
+.. code::
+
+    tensor([0.0009, 0.0011, 0.0010, 0.0011, 0.0034, 0.0013, 0.0010, 0.0010, 0.0013,
+            0.0012, 0.0011, 0.0010, 0.0010, 0.0010, 0.0010, 0.0010, 0.0009, 0.0015,
+            0.0016, 0.0036, 0.0012, 0.0009, 0.0010, 0.0014, 0.0008, 0.0008, 0.0008,
+            ...,
+            0.0019, 0.0023, 0.0013, 0.0018, 0.0012, 0.0031, 0.0015, 0.0013, 0.0014,
+            0.0022, 0.0011, 0.0024])
+
+Zero-point tensor -
+
+.. code::
+
+    tensor([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+            ..,
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+           dtype=torch.int32)
+
+4. Comparing Results with Eager Mode
+------------------------------------
+
+Following results show the F1 score and model size for Eager Mode Quantization of the same model by following the steps mentioned in the `tutorial <https://pytorch.org/tutorials/intermediate/dynamic_quantization_bert_tutorial.html#evaluate-the-inference-accuracy-and-time>`_. Results show that Eager and Graph Mode Quantization on the model produce identical results.
+
+.. code::
+
+    FP32 model results -
+    Size (MB): 438.016605
+    'f1': 0.901
+
+    INT8 model results -
+    Size (MB): 182.878029
+    'f1': 0.902
+
+5. Benchmarking the Model
+-------------------------
+
+We benchmark the model with dummy input and compare the Float model with Eager and Graph Mode Quantized Model on a production server machine.
+
+.. code:: python
+
+    def benchmark(model):
+        model = torch.jit.load(model)
+        model.eval()
+        torch.set_num_threads(1)
+        input_ids = ids_tensor([8, 128], 2)
+        token_type_ids = ids_tensor([8, 128], 2)
+        attention_mask = ids_tensor([8, 128], vocab_size=2)
+        elapsed = 0
+        for _i in range(50):
+            start = time.time()
+            output = model(input_ids, token_type_ids, attention_mask)
+            end = time.time()
+            elapsed = elapsed + (end - start)
+        print('Elapsed time: ', (elapsed / 50), ' s')
+        return
+    print("Running benchmark for Float model")
+    benchmark(args.jit_model_path_float)
+    print("Running benchmark for Eager Mode Quantized model")
+    benchmark(args.jit_model_path_eager)
+    print("Running benchmark for Graph Mode Quantized model")
+    benchmark(args.jit_model_path_graph)
+
+.. code::
+
+    Running benchmark for Float model
+    Elapsed time: 4.49 s
+    Running benchmark for Eager Mode Quantized model
+    Elapsed time: 2.67 s
+    Running benchmark for Graph Mode Quantized model
+    Elapsed time: 2.69 s
+    As we can see both graph mode and eager mode quantized model have a similar speed up over the floating point model.
```

Comment:
Suggestion: Add a conclusion section with links to references or learn more at the end. Example:
https://pytorch.org/tutorials/intermediate/dynamic_quantization_bert_tutorial.html#conclusion